syntax = "proto3";

package hello;

// 서비스 정의
service Greeter {
    rpc SayHello (HelloRequest) returns (HelloReply) {}
    rpc ReqRemoteCommand (RcmdRequest) returns (RcmdReply) {}
}

// 요청 메시지
message HelloRequest {
    string name = 1;
}

// 응답 메시지
message HelloReply {
    string message = 1;
}

message RcmdRequest {
    string command = 1;
    string param = 2;
}

message RcmdReply {
    string ack = 1;
}